### DISCUSSION TITLE
请问可不可以加上每个停贷的楼盘总共有多少户或者房子啊？

### DISCUSSION THREAD

**Discussion by Wtt0413** [WeNeedHome/SummaryOfLoanSuspension#876] [Ideas]
如题，大概的也行，方便用来估算规模
只是一个小建议
谢谢

**Comment by imhappyok** [comment]
建议不错，不过这些详细数据，有人能统计吗，可能只有每个楼盘的倡导人才能初步统计然后发上来进一步统计，如果300多个楼盘，一个楼盘3000户左右，加起来解决100w户，最大影响可能在500w人左右，但是不是每户都停贷，不知道有没有1/3

**Comment by imhappyok** [comment]
昨天好像看到一个新闻据说涉及贷款金额3800亿左右，按照一户平均欠贷70w，估计大概涉及50w户左右吧，都是个人猜测的数据，不能说明什么

**Reply by Wtt0413** [reply]
谢谢！再多问一句，请问你还记得是哪个新闻吗？

**Reply by imhappyok** [reply]
不记得了，反正大概意思就是说3000亿而已，对于房贷总额38w亿来说不到，不到1%，影响不大，昨天就回复你了，咋看不到。。。

**Reply by Wtt0413** [reply]
啊，我看到了，就是搜不到你说的那个新闻，所以再问一句你记不记得是哪个新闻说的。。。不过总之谢谢你啦！

**Comment by MarkShawn2020** [comment]
可以，后端接口后续会跟上，与楼盘价格一起

**Reply by Wtt0413** [reply]
麻烦了！感谢！！！

**Reply by MarkShawn2020** [reply]
> 麻烦了！感谢！！！

请见 #950

**Reply by Wtt0413** [reply]
谢谢！看到了！我会一直关注的！很感谢你一直对数据库的贡献！可惜我代码不好帮不上什么忙，不过希望有其他人可以帮忙

**Reply by pinerose99** [reply]
> 可以，后端接口后续会跟上，与楼盘价格一起

太好了，非常感谢！
